.github/actions/e2e-arc-test/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'E2E ARC Test Action'
+description: 'Includes common arc installation, setup and test file run'
+
+inputs:
+  github-token:
+    description: 'A Github PAT'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Helm installation
+      run: |
+        curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+        sudo apt-get install apt-transport-https --yes
+        echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+        sudo apt-get update
+        sudo apt-get install helm
+      shell: bash
+    - name: Install ARC
+      run: helm install arc --namespace "arc-systems" --create-namespace oci://ghcr.io/actions/actions-runner-controller-charts/actions-runner-controller-2 --version 0.1.0
```

Comment:
Not sure, was following the commands from here https://github.com/actions/actions-runner-controller/tree/master/docs/preview/actions-runner-controller-2
Can't we just take the `latest` instead of specifying the version?

Reply:
@nikola-jokic had a good observation, we should use the local charts to test the code before publishing a new version, updated

Reply:
Yeah I agree, that's why making this an input parameter might be a good idea. We can then instruct it to be `local` or specify a release version `x.x.x` depending on the usecase